ReactAndroid/src/main/java/com/facebook/react/modules/location/LocationModule.java

```diff
@@ -104,6 +104,11 @@ private static LocationOptions fromReactMap(ReadableMap map) {
     }
   }
 
+  @ReactMethod
```

Comment:
Remove this completely and instead check on the JS side if the method exists before calling it.